panel/lxqtpanel.h

```diff
@@ -174,6 +174,7 @@ private slots:
 
     ILXQtPanel::Position mPosition;
     int mScreenNum;
+    int mActualScreenNum;
```

Comment:
I know it's easy to figure out, but I miss some comments makeing clear why the need for the two similar variables.